CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## master (unreleased)
 
+## 1.4.1 (2024-08-20)
```

Comment:
It's better to not touch versions/dates in PRs—you never know when it's gonna be merged 🙂